CHANGELOG.next.asciidoc

```diff
@@ -210,6 +210,7 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 - Add `redisenterprise` module. {pull}16482[16482] {issue}15269[15269]
 - Align fields to ECS and add more tests for the azure module. {issue}16024[16024] {pull}16754[16754]
 - Add PubSub metricset to Google Cloud Platform module {pull}15536[15536]
+- Add overview dashboard for googlecloud compute. {issue}16534[16534] {pull}16819[16819]
```

Comment:
I think both changelog entries should be consistent :)

Reply:
😂 I prefer googlecloud module but I don't want to modify the previous changelog entry 😬